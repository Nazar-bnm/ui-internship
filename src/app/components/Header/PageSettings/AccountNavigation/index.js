import React from 'react';
import PropTypes from 'prop-types';


const AccountNavigation = (props) => {
  return (
    <div className={props.grid}>
      <span>my account</span>
      <span>wish list</span>
      <span>Checkout</span>
      <span>log in</span>
    </div>
  );
};

AccountNavigation.propTypes = {
  grid: PropTypes.string,
};

export default AccountNavigation;
