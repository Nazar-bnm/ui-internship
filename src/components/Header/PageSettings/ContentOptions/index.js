import ContentOptions from './ContentOptions';

export default ContentOptions;
