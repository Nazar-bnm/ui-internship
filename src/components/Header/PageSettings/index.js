import PageSettings from './PageSettings';

export default PageSettings;
