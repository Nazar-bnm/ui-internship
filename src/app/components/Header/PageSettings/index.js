import React from 'react';
import ContentOptions from './ContentOptions';
import AccountNavigation from './AccountNavigation';

const PageSettings = () => (
  <div className="content container">
    <ContentOptions grid="col-4"/>
    <h3 className="col-3 col-center">
        free shipping on orders above 50$
    </h3>
    <AccountNavigation grid="col-4"/>
  </div>
);

export default PageSettings;
