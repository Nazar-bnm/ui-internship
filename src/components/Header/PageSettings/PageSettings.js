import React from 'react';
import cx from 'classnames';
import ContentOptions from './ContentOptions';
import AccountNavigation from './AccountNavigation';

export const CN = 'page-settings';

const PageSettings = () => (
  <div className={cx(CN, 'content container')}>
    <ContentOptions />
    <h3 className={`${CN}__heading heading`}>
        free shipping on orders above 50$
    </h3>
    <AccountNavigation />
  </div>
);

export default PageSettings;
