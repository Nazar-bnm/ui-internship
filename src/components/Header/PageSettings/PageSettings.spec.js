import React from 'react';
import { shallow } from 'enzyme';

import PageSettings from './PageSettings';

describe('<PageSettings />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <PageSettings />
    );
  });
  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
