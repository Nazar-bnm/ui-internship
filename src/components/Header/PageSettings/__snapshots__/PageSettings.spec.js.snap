// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageSettings /> should match snapshot 1`] = `
<div
  className="content container"
>
  <ContentOptions />
  <h3
    className="heading"
  >
    free shipping on orders above 50$
  </h3>
  <AccountNavigation />
</div>
`;
